packages/api/internal/orchestrator/orchestrator.go

```diff
@@ -35,6 +35,10 @@ const statusLogInterval = time.Second * 20
 var ErrNodeNotFound = errors.New("node not found")
 
 type Orchestrator struct {
+	// Deprecated: Nomad knows which port the orchestrator is listening on. Keep this
```

Comment:
We also need to track proxy port. It will ve needed for sandbox traffic routing done by client proxy.

Data are send via Redis sandbox catalog done in API. We are now sending just IP to orchestrator job

Reply:
Definitely! I was hoping to keep this PR smaller by omitting those changes, but I can extend this to include those changes as well.